modules/module-info.php

```diff
@@ -404,12 +404,29 @@ function jetpack_photon_more_link() {
 
 function jetpack_photon_more_info() {
 	esc_html_e(
-		"Your images are automatically optimized for different display resolutions to serve the best
-		possible image quality. We also cache and serve them from our fast, global network (CDN)."
+		'Jetpack will optimize your images and serve them from the server location nearest
+		to your visitors. Using our global \'content delivery network\' will boost the loading speed of your site.'
```

Comment:
I don't think we need the single quotes around content delivery network since we're not talking about that word, or being sarcastic about it, or making a reference of it inside a bigger context.
If the intention here is to highlight it, quotes are not highlight devices and italic or bold should be used instead.

Reply:
I agree. Maybe instead we could just say "Using our global network..."

Reply:
I'm impartial. I was copying how @rickybanister had laid out his copy in the referenced issue. Glad to change if we come to a consensus on what to change it to.

Reply:
Agreed with @eliorivero on removing the quotes.